conf/prod.routes

```diff
@@ -1,6 +1,9 @@
 # Add all the application routes to the app.routes file
```

Comment:
can we a copy of this file and call it something like `private_beta.routes` . Then we can update app-config-prod to point to `private_beta.routes` and everywhere else would use `prod.routes` by default. `returns.routes` wouldn't be included in `private_beta.routes`.

Reply:
changed - if I remember correctly this is not something that is recommended to do as it might break something in the deployment but I can't see how and I can't recall the exact reason. Only one way to find out....

Reply:
let's try pointing it to `private_beta.routes` in staging and see if it breaks anything before making the same change to prod then to double check - hopefully it just works